fhirzeug/fhirspec.py

```diff
@@ -1194,8 +1200,5 @@ def __init__(self, mapping_arr):
         pass
 
 
-def _is_string(element):
-    isstr = isinstance(element, str)
-    if not isstr and sys.version_info[0] < 3:  # Python 2.x has 'str' and 'unicode'
-        isstr = isinstance(element, basestring)
-    return isstr
+def _is_string(element) -> bool:
```

Comment:
actually we could also remove the function completely and replace the `_is_string` occurrences in the code....